src/frontpage/components/Articles/SmallArticle.tsx

```diff
@@ -1,13 +1,12 @@
 import { IArticle } from 'articles/models/Article';
 import classnames from 'classnames';
 import ResponsiveImage from 'common/components/ResponsiveImage';
-import { DOMAIN } from 'common/constants/endpoints';
 import React from 'react';
 import style from './articles.less';
 
-const SmallArticle = ({ absolute_url, heading, image, ingress_short }: IArticle) => {
+const SmallArticle = ({ heading, image, ingress_short, id }: IArticle) => {
   return (
-    <a href={DOMAIN + absolute_url}>
+    <a href={'articles/' + id}>
```

Comment:
In this case you can import `routes` from `ArticlesRouter`, and use `routes.detail`